lib/phonebook/ui/pages/association_page/association_page.dart

```diff
@@ -0,0 +1,137 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:heroicons/heroicons.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:myecl/phonebook/providers/association_kind_provider.dart';
+import 'package:myecl/phonebook/providers/association_member_sorted_list_provider.dart';
+import 'package:myecl/phonebook/providers/association_picture_provider.dart';
+import 'package:myecl/phonebook/providers/association_provider.dart';
+import 'package:myecl/phonebook/providers/association_member_list_provider.dart';
+import 'package:myecl/phonebook/providers/phonebook_admin_provider.dart';
+import 'package:myecl/phonebook/router.dart';
+import 'package:myecl/phonebook/tools/constants.dart';
+import 'package:myecl/phonebook/ui/pages/association_page/member_card.dart';
+import 'package:myecl/phonebook/ui/pages/association_page/web_member_card.dart';
+import 'package:myecl/phonebook/ui/phonebook.dart';
+import 'package:myecl/tools/ui/builders/async_child.dart';
+import 'package:myecl/tools/ui/layouts/refresher.dart';
+import 'package:qlevar_router/qlevar_router.dart';
+
+class AssociationPage extends HookConsumerWidget {
+  const AssociationPage({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final association = ref.watch(associationProvider);
+    final associationMemberList = ref.watch(associationMemberListProvider);
+    final associationMemberSortedList =
+        ref.watch(associationMemberSortedListProvider);
+    final associationMemberListNotifier =
+        ref.watch(associationMemberListProvider.notifier);
+    final associationPictureNotifier =
+        ref.watch(associationPictureProvider.notifier);
+    final isPresident = ref.watch(isAssociationPresidentProvider);
+    final kindNotifier = ref.watch(associationKindProvider.notifier);
+
+    return PhonebookTemplate(
+      child: Refresher(
+        onRefresh: () async {
+          await associationMemberListNotifier.loadMembers(
+            association.id,
+            association.mandateYear.toString(),
+          );
```

Comment:
Shouldn't we refresh the association's information too ?

Reply:
fixed